components/hab/src/cli.rs

```diff
@@ -546,6 +547,15 @@ pub fn get() -> App<'static, 'static> {
                 )
             )
         )
+        (@subcommand sup =>
+            (@setting VersionlessSubcommands)
+            (@setting SubcommandRequiredElseHelp)
+            // these are the only sup cmds handled in `hab`, the remaining
+            // supervisor related commands are handled in `hab-sup`
+            (subcommand: sub_sup_depart().aliases(&["d", "de", "dep", "depa", "depart"]))
+            (subcommand: sub_sup_secret().aliases(&["sec", "secr"]))
```

Comment:
Similar to `depart`, `secret` was previously undocumented, but I'm not sure if that was intention or not. @christophermaier, what do you think?

Reply:
Yup, `secret` is supposed to be documented, too; see #5363